sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -1904,16 +1904,16 @@ the `ext_oneapi_signal_external_semaphore` call complete.
 are non-blocking, asynchronous operations.
 
 The user must ensure to destroy all external semaphore objects once they are no 
```

Comment:
```suggestion
The user must ensure to release all external semaphore objects once they are no 
```
?

Reply:
I've updated the wording in the spec as it might cause confusion as to what exactly is being destroyed.